### PR TITLE
Updated dependencies + code to Purescript 0.9

### PR DESCRIPTION
**Pull request by PipocaQuemada**

### CODE REVIEW

File: src/Data/Version/Haskell.purs [hunk 1]
```diff
@@ -9,24 +9,24 @@
 -- | though, you should probably be using `Data.Version`.
 module Data.Version.Haskell where
 
-import Prelude
+import Prelude (class Show, class Ord, class Eq, show, compare, flip, pure, bind, map, (<>), (==), (&&), (<<<), ($), (*>), (||), (<$>))
```
Comment (hdgarrood) [inline src/Data/Version/Haskell.purs]: Can we keep the Prelude import implicit here? IIRC you're allowed one implicit import before you start getting warnings.

**Comment by hdgarrood** [comment]
It looks like the CI is failing because of an old version of either Pulp or purescript (I haven't looked closely). Also, we should stop testing on node.js versions <4 at the same time, as I don't think Pulp supports them any more, and I'm fairly sure quite a few other parts of the PS ecosystem no longer do either.

Other than that, looks great, thanks!

**Comment by PipocaQuemada** [comment]
I removed the old versions of node, and fixed up travis to install and use bower.

Currently, it's timing out after getting to

> pulp build
> - Building project in /home/travis/build/hdgarrood/purescript-versions

Since nothing's printed for the next 10 minutes.  That seems rather long for compilation to take; any idea of what's going wrong?

**Comment by hdgarrood** [comment]
I think this is something we've seen a few times before, we think the cause is that Travis reports that it has more cores available than it actually does. Try adding `dist: trusty` and `sudo: required` to .travis.yml. You could look at prelude's .travis.yml as an example.

**Comment by PipocaQuemada** [comment]
Looks like travis builds, now.  Thanks!

**Comment by hdgarrood** [comment]
Great, thanks!

**Comment by hdgarrood** [comment]
I've just tagged 2.0.0. Could you please check everything works and publish to Pursuit, if you can? I don't actually have access to a functioning PC at the moment.

**Pull request merged**